test/Error/outofmem.js

```diff
@@ -3,18 +3,7 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
-try
-{
-    var longString = "A";
-    for (var i = 0; i < 31; i++)
-        longString += longString;
-    WScript.Echo(longString.toString());
-}
-catch (e)
-{
-    WScript.Echo(e.name);
-    WScript.Echo(e.message);
-    WScript.Echo(e.description);
-    WScript.Echo(e.number);
-}
-
+var longString = "A";
+for (var i = 0; i < 31; i++)
+   longString += longString;
+WScript.Echo(longString.toString());
```

Comment:
I think the idea was to expand this test, not replace it. The existing checks work just fine and don't cause the test to crash. After the test code that was here previously, you can append this code here, and the `Error: Out of memory` message on the final line of the baseline.

That way we test what this test was originally testing as well as the new case you've added to display the `Error: Out of memory` message.